modules/core/src/main/java/org/apache/synapse/message/store/impl/commons/MessageConverter.java

```diff
@@ -131,6 +132,10 @@ public static MessageContext toMessageContext(StorableMessage message,
                 axis2Ctx.setOperationContext(operationContext);
                 axis2Ctx.setAxisService(axisService);
                 axis2Ctx.setAxisOperation(axisOperation);
+            } else {
+                if (logger.isDebugEnabled()) {
```

Comment:
True. But as a practice, it is better to keep it in this way, IMO. Anyway this condition will be checked when the logger.debug() executes, even we remove the "if (logger.isDebugEnabled())". Therefore, it does not create any difference. What do you think?